pkg/utils/transform/transform.go

```diff
@@ -69,12 +69,30 @@ func InjectNamespaceConditional(preserveNamespace, targetNamespace string) mf.Tr
 	}
 }
 
+func InjectNamespaceRoleBindingConditional(preserveNS, preserveRBSubjectNS, targetNamespace string) mf.Transformer {
```

Comment:
let us make this function check only `preserveRBSubjectNS` check
and reuse `InjectNamespaceRoleBindingSubjects` for `preserveNamespace Check

Reply:
I didn't understand this. Why should we check only one thing?

Reply:
I didn't get this. So we should have only one check there?

Reply:
https://github.com/openshift/tektoncd-pipeline-operator/pull/393/files?file-filters%5B%5D=.go#r467888300

Reply:
it makes sense now